testing/trino-server-dev/etc/catalog/delta.properties

```diff
@@ -0,0 +1,10 @@
+connector.name=delta-lake
+
+# Configuration appropriate for Hive as started by product test environment, e.g.
+#   trino-product-tests-launcher/bin/run-launcher env up --environment singlenode --without-trino
+# On Mac, this additionally requires that you add "<your external IP> hadoop-master" to /etc/hosts
```

Comment:
the env uses MinIO so is this line needed?
or do i need anything else in the /etc/hosts ?